Xamarin.PropertyEditing.Mac/Controls/CollectionEditorWindow.cs

```diff
@@ -26,19 +26,28 @@ public CollectionEditorWindow (IHostResourceProvider hostResources, CollectionPr
 
 			ContentView.AddSubview (this.collectionEditor);
 
-			this.ok = NSButton.CreateButton (Properties.Resources.OK, OnOked);
-			this.ok.AccessibilityEnabled = true;
-			this.ok.AccessibilityTitle = Properties.Resources.AccessibilityCollectionOKButton;
-			this.ok.Highlighted = true;
-			this.ok.TranslatesAutoresizingMaskIntoConstraints = false;
-			
-			//this.ok.KeyEquivalent = "\r"; // FIXME: The type selector popup doesn't eat this key, so it ends up closing both.
```

Comment:
If you're going to uncomment/reintroduce a bit of code that has an explanation as to why it's commented out, be sure to re-test the stated issue. In this case, this broken behavior (pressing enter while the type selector is open closes the entire dialog) is now present again.

Reply:
Fixed.